storybook/src/docs/borders.docs.mdx

```diff
@@ -2,9 +2,9 @@
 
 import { Meta } from "@storybook/blocks";
 
-<Meta title="Docs/Design Guidelines/Borders" />
+<Meta title="Foundation/Design Tokens/Border" />
 
-# Borders
+# Border
 
 ## Widths
 
```

Comment:
Do we want to use the same directory structure in `/src/docs`? It's quite a long list now

Reply:
Yes, as soon as the hierarchy and names are final.